.circleci/config.yml

```diff
@@ -635,40 +680,69 @@ workflows:
                 - services/shuttle-tide
                 - services/shuttle-tower
                 - services/shuttle-warp
-      - check-standalone:
-          # shuttle-shared-db has mutually exclusive features
-          # so we run checks for each feature separately
+      - test-standalone:
+          # Has mutually exclusive features, so we run checks for each feature separately
           name: "resources/shared-db: << matrix.features >>"
           matrix:
-            alias: check-standalone-shared-db
+            alias: test-standalone-shared-db
             parameters:
-              path: [resources/shared-db]
+              path:
+                - resources/shared-db
               features:
                 - "-F mongodb"
                 - "-F postgres"
                 - "-F postgres-rustls"
-      - platform-test:
+      ############ This crate has no tests, therefore excluded.
+      # - test-standalone:
+      #     # Has mutually exclusive features, so we run checks for each feature separately
+      #     name: "resources/aws-rds: << matrix.features >>"
+      #     matrix:
+      #       alias: test-standalone-aws-rds
+      #       parameters:
+      #         path:
+      #           - resources/aws-rds
+      #         features:
+      #           - "-F postgres"
+      #           - "-F postgres-rustls"
+      #           - "-F mysql"
+      #           - "-F mysql-rustls"
+      #           - "-F mariadb"
+      #           - "-F mariadb-rustls"
```

Comment:
Should we remove this then?

Reply:
I'm intending to have that comment as a "how it would be done if it was needed".
Or is tests in that crate irrelevant and this comment too bloated?

Reply:
I think looking at the test-standalone for shared db makes it easy to understand how to test rds as well if we add tests in the future, so I don't think we need this.